src/plugins/twitter/twitter.js

```diff
@@ -141,16 +135,26 @@ var componentName = "wb-twitter",
 					childList: true,
 					subtree: true
 				} );
-			}
 
-			Modernizr.load( {
-				load: ( protocol.indexOf( "http" ) === -1 ? "http:" : protocol ) + "//platform.twitter.com/widgets.js",
-				complete: function() {
+				// Handle the case where the iframe never loads - wait 5 seconds before displaying a fallback message
+				setTimeout( () => {
+					const iframe = eventTarget.querySelector( "iframe.twitter-timeline" );
+					if ( !iframe ) {
+						const fallbackMessage = "Twitter timeline is currently unavailable.";
+						console.warn( componentName + ": " + fallbackMessage );
+						observer.disconnect();
+					}
+				}, 5000 ); // 5 seconds
 
-					// Identify that initialization has completed
-					wb.ready( $( eventTarget ), componentName );
-				}
-			} );
+			}
+
+			// Load the Twitter widget script
+			const script = document.createElement( "script" );
+			script.src = ( protocol.indexOf( "http" ) === -1 ? "http:" : protocol ) + "//platform.twitter.com/widgets.js";
+			script.onload = function() {
+				wb.ready( $( eventTarget ), componentName );
+			};
+			document.head.appendChild( script );
```

Comment:
This logic seems to add a the script to the head everytime the plugin is called. Please wrap this in a condition whether the script has already been loaded or not.